app/lib/redis.js

```diff
@@ -2,13 +2,11 @@
 
 const Redis = require('ioredis');
 
-Redis.Promise.onPossiblyUnhandledRejection((err) => console.log(`Unhandled Redis error: ${err}`));
```

Comment:
Why did you remove this? I think it catches unhandled promises and prevents the app from crashing?